tests/test_lstm_policy.py

```diff
@@ -31,7 +38,33 @@ def __init__(self, sess, ob_space, ac_space, n_env, n_steps, n_batch, n_lstm=64,
                          layer_norm=True, feature_extraction="mlp", **_kwargs)
 
 
+class CartPoleNoVelEnv(CartPoleEnv):
+    """Variant of CartPoleEnv with velocity information removed. This task requires memory to solve."""
+
+    def __init__(self):
+        super().__init__()
```

Comment:
```suggestion
        super(CartPoleNoVelEnv, self).__init__()
```

I don't know if this is for this reason that codacy complains...